rating-service/src/main/java/edu/miu/ratingservice/controller/RatingController.java

```diff
@@ -0,0 +1,71 @@
+package edu.miu.ratingservice.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.miu.ratingservice.dto.request.RatingRequestDTO;
+import edu.miu.ratingservice.service.RatingService;
+import feign.Response;
+
+import static edu.miu.ratingservice.constant.WebResourceKeyConstant.ALL;
+import static edu.miu.ratingservice.constant.WebResourceKeyConstant.API_V1;
+import static edu.miu.ratingservice.constant.WebResourceKeyConstant.RatingConstants.RATING_BASE;
+
+@RestController
+@RequestMapping(value = API_V1 + RATING_BASE)
+public class RatingController {
+    private final RatingService ratingService;
+
+    public RatingController(RatingService ratingService) {
+        this.ratingService = ratingService;
+    }
+
+    // // list
+    // @GetMapping("/meal/{id}")
+    // public ResponseEntity<?> getRatingsByMeal(Long id) {
+    // return null;
+    // }
+
+    // // list return type
+    // @GetMapping
+    // public ResponseEntity<?> getRatingsByDay() {
+    // return null;
+    // }
+
+    // // List return type
+    // @GetMapping("/user/rating/{id}")
+    // public ResponseEntity<?> getRatingsByUser(Long id) {
+    // return null;
+    // }
+
+    // String return type
+    @GetMapping("/{id}")
```

Comment:
See if we need another endpoint like getting the mean rating for a specific meal or something like that.